frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx

```diff
@@ -21,7 +21,7 @@ export const CreateStorageSystemHeader: React.FC<CreateStorageSystemHeaderProps>
         <Text component={TextVariants.h1}>{t('ceph-storage-plugin~Create StorageSystem')}</Text>
         <Text component={TextVariants.small}>
           {t(
-            'ceph-storage-plugin~StorageSystem is an entity of OpenShift Data Foundation. It represents all of the required storage and compute resources.',
+            'ceph-storage-plugin~Create a storage system to represent your OpenShift Data Foundation system and all its required storage and computing resources.',
```

Comment:
```suggestion
            'ceph-storage-plugin~Create a StorageSystem to represent your OpenShift Data Foundation system and all its required storage and computing resources.',
```
Not sure, but will it be StorageSystem ??

Reply:
This is what I found in PatternFly > Capitalization doc:

"Capitalize Red Hat feature names when they’re used as proper nouns or when they refer to a capitalized UI term (like a navigation item). **Write them in lowercase when they’re used to describe generic concepts.**"
This text is present in page heading under `Create StorageSystem` and i felt that it is describing what we will be doing. Not sure myself which is correct. WDYT ??

Reply:
Makes sense then.

Reply:
Is `StorageSystem` a resource kind?

Reply:
Yes, it is a resource of `kind: 'StorageSystem'`.

Reply:
We've used the CamelCase name in admin console in similar contexts (i.e. "Create a new ImageStreamTag by pushing an image...").